plexapi/media.py

```diff
@@ -1106,3 +1106,29 @@ def _loadData(self, data):
     @deprecated('use "languageCodes" instead')
     def languageCode(self):
         return self.languageCodes
+
+
+class Theme(PlexObject):
```

Comment:
Move up after `Poster`.

Reply:
Actually just subclass `BaseImage` with a different `TAG`.

Reply:
> Actually just subclass `BaseImage` with a different `TAG`.

There is no `provider` or `thumb` for this. Is it still preferable to subclass it as `BaseImage`?

Reply:
There actually is a `thumb`.

Yes, subclass. `provider` will just be `None`.